### DISCUSSION TITLE
Welcome to .NET MAUI Discussions!

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please re-familiarize yourself with our [.NET Foundation Code of Conduct](https://dotnetfoundation.org/about/code-of-conduct).

We will be moving Issues that are more discussion-like to this new feature, and should something become a spec we can link them.

For Xamarin.Forms related discussions, see https://github.com/xamarin/xamarin.forms/discussions.

**Comment by Happypig375** [comment]
GitHub Discussions is out of Beta? Great news! Let me enable it for my repositories too.

**Reply by Happypig375** [reply]
Wait... I cannot find the setting. How can I enable it?
